source/basic.tex

```diff
@@ -4257,98 +4214,175 @@
+the range exponent of that type has padding bits; see \ref{basic.types}.
+\end{note}
 
 \pnum
+\indextext{\idxcode{wchar_t}|see{type, \tcode{wchar_t}}}%
+\indextext{type!\idxcode{wchar_t}}%
+\indextext{underlying type|see{type, underlying}}%
+\indextext{type!underlying!\idxcode{wchar_t}}%
+Type \tcode{wchar_t} is a distinct type that has
+an \impldef{underlying type of \tcode{wchar_t}}
+signed or unsigned integer type as its underlying type.
+The values of type \tcode{wchar_t} can represent
+distinct codes for all members of the largest extended character set
+specified among the supported locales\iref{locale}.
 \indextext{\idxcode{char16_t}|see{type, \tcode{char16_t}}}%
 \indextext{\idxcode{char32_t}|see{type, \tcode{char32_t}}}%
-\indextext{\idxcode{wchar_t}|see{type, \tcode{wchar_t}}}%
 \indextext{type!\idxcode{char16_t}}%
 \indextext{type!\idxcode{char32_t}}%
-\indextext{type!\idxcode{wchar_t}}%
-\indextext{underlying type|see{type, underlying}}%
 \indextext{type!underlying!\idxcode{char16_t}}%
 \indextext{type!underlying!\idxcode{char32_t}}%
-Type \tcode{wchar_t} is a distinct type whose values can represent
-distinct codes for all members of the largest extended character set
-specified among the supported locales\iref{locale}. Type
-\tcode{wchar_t} shall have the same size, signedness, and alignment
-requirements\iref{basic.align} as one of the other integral types,
-called its \defnx{underlying type}{type!underlying!\idxcode{wchar_t}}. Types \tcode{char16_t} and
-\tcode{char32_t} denote distinct types with the same size, signedness,
-and alignment as \tcode{uint_least16_t} and \tcode{uint_least32_t},
-respectively, in \tcode{<cstdint>}, called the underlying types.
+Types \tcode{char16_t} and \tcode{char32_t} denote distinct types
+whose underlying types are \tcode{uint_least16_t} and \tcode{uint_least32_t},
+respectively, in \tcode{<cstdint>}.
 
 \pnum
 \indextext{Boolean type}%
-Values of type \tcode{bool} are either \tcode{true} or
-\tcode{false}.\footnote{Using a \tcode{bool} value in ways described by this document
-as ``undefined'', such as by examining the value of an
-uninitialized automatic object, might cause it to behave as if it is
-neither \tcode{true} nor \tcode{false}.}
-\begin{note} There are no \tcode{signed}, \tcode{unsigned}, \tcode{short},
-or \tcode{long bool} types or values. \end{note} Values of type
-\tcode{bool} participate in integral promotions\iref{conv.prom}.
+\indextext{type!Boolean}%
+Type \tcode{bool} is a distinct type that has
+the same object representation,
+value representation, and
+alignment requirements as
+an \impldef{underlying type of \tcode{bool}} unsigned integer type.
+The values of type \tcode{bool} are
+\tcode{true} and \tcode{false}.\footnote{Using a \tcode{bool} value
+in ways described by this document as ``undefined'',
+such as by examining the value of an uninitialized automatic object,
+might cause it to behave as if it is neither \tcode{true} nor \tcode{false}.}
```

Comment:
Nitpick on the note: it can behave as neither true nor false, but it can also behave as both true and false.

```suggestion
might cause it to behave as if it is neither \tcode{true} nor \tcode{false},
or might cause it to behave as if it is both \tcode{true} and \tcode{false}.}
```

Reply:
@jfbastien Please open a separate editorial issue on this if you'd like to see this clarified - even better if you can suggest new wording :)

Reply:
I suggested wording right there in GitHub, you can click "accept" and we're done. I can't suggest new wording right now without causing a merge conflict with this PR.

Reply:
@jfbastien please open a new issue and we can merge it later?  This is existing wording so it should not be handled as part of this paper.  Thanks for understanding!

Reply:
https://github.com/cplusplus/draft/issues/2483